build/yaml/crd/nsx.vmware.com_ippools.yaml

```diff
@@ -0,0 +1,125 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: ippools.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: IPPool
+    listKind: IPPoolList
+    plural: ippools
+    singular: ippool
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: IPPool is the Schema for the ippools API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: IPPoolSpec defines the desired state of IPPool.
+            properties:
+              subnets:
+                description: Subnets defines set of subnets need to be allocated.
+                items:
+                  description: SubnetRequest defines the subnet allocation request.
+                  properties:
+                    ipFamily:
+                      description: IPFamily defines the IP family type for this subnet,
+                        could be IPv4 or IPv6. This is optional, the default is IPv4.
+                      type: string
+                    name:
+                      description: Name defines the name of this subnet.
+                      type: string
+                    prefixLength:
+                      description: PrefixLength defines prefix length for this subnet.
+                      type: integer
+                  required:
+                  - name
+                  type: object
+                type: array
+              type:
+                description: Type defines the type of this IPPool, public or private.
+                type: string
+            required:
+            - subnets
```

Comment:
subnets and type should be optional, right? @heypnus

Reply:
ok